Cargo.toml

```diff
@@ -18,11 +18,6 @@ log = "0.3"
 protobuf = "1.2"
 quick-error = "0.2"
 rand = "0.4"
-clippy = {version = "*", optional = true}
```

Comment:
you can use `clippy = {version = "^0", optional = true}` instead to pass the crate like rust prometheus does.

Reply:
I prefer the current way. So `allow_failure` is unnecessary anymore.

Reply:
how can we use clippy in local machine now?

Reply:
```
- cargo +nightly-2018-01-12 install -f clippy
- cargo +nightly-2018-01-12 clippy
```

Reply:
Or you can just do nothing and let CI lints your code.

Reply:
Got it